requirement-checker/src/Printer.php

```diff
@@ -148,6 +148,14 @@ public function printv(string $message, int $verbosity, ?string $style = null):
             $this->supportColors ? $this->styles['reset'] : ''
         );
 
-        echo $message;
+        if (getenv('BOX_REQUIREMENTS_CHECKER_LOG_TO_STDOUT') === '1')
+        {
```

Comment:
let's keep to the traditional CS of the project:

```php
if (smth) {
    // ...
} else {
    // ...
}
```

Sorry I don't remember if `make cs` fixes this or not (I don't think it is applied to the requirement checker at all)

Reply:
CS is updated.